### DISCUSSION TITLE
it been more than 5 day's but i didnt get my benefit till now

### DISCUSSION THREAD

**Discussion by AYUSH-pro-grammer** [community/community#146225] [GitHub Education]
### Hi everyone,

it been more than 5 day's but i didnt get my benefit till now

![image](https://github.com/user-attachments/assets/591aa270-279a-4c4f-bf18-07d343e2257f)

**Comment by AmazingK2k3** [comment]
Hi , I am facing the same issue. Can anyone do something about it?

**Comment by Aygyun951** [comment]
Are you still facing the same issue? I have the same issue I don't believe that it should take that long when it's not the start of the semester and  if we are approved we should be able to get this fast enough.

**Reply by AYUSH-pro-grammer** [reply]
yes bro issue still here

**Comment by davevad93** [comment]
Hi @AYUSH-pro-grammer ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @AYUSH-pro-grammer was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: 
#132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287, #145640, #145995

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
